### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by donno2048**

### CODE REVIEW

**Comment by Geolykt** [comment]
Some of the code you removed are remnants of old code, where as I have no real Idea what the other half does; I'll be neutral on this PR unless other things happen here

**Comment by hydrostaticcog** [comment]
A couple of things:

- This PR was created on the master branch and targets the master branch. Please target the develop branch. You should use the develop branch to make PRs so that you have the most recent code.

- This PR will interfere with #68.

As for everything else, I'm not sure how much of these changes we need since some the files you have changed are not in use yet, and I'm not sure how future contributors will use the world generator for example. The logging catch in Multiprocessing.py is a good catch though.

File: classes/mcPy/MultiProcessing.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def add_task(self, func, args: list, **kwargs):
         try:
             self.TASK_LIST.put_nowait(data)
         except Full:
-            logging.warning()
+            logging.warning(Full)
```
Comment (hydrostaticcog) [inline classes/mcPy/MultiProcessing.py:61]: Good catch, though we might want to make this a little cleaner (add a better message than just the error)

File: classes/WorldGenerator.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from math import floor, sqrt
 from random import randint
 
-from .BasicClasses import Block, Chunk, Region
```
Comment (hydrostaticcog) [inline classes/WorldGenerator.py:5]: We should leave this import, as it could be useful in the future.

File: libs/configparser/config.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def __enter__(self):
         self.load_config()
         return self
 
-    def __exit__(self):
+    def __exit__(self, type, value, traceback):
```
Comment (hydrostaticcog) [inline libs/configparser/config.py:38]: This file is moved in `develop`, so please remove this.

**Review by hydrostaticcog — CHANGES_REQUESTED** [review]
Some issues with conflicts with `develop` branch, and some things that could be improved upon.

**Comment by donno2048** [comment]
> We should leave this import, as it could be useful in the future.
> This file is moved in `develop`, so please remove this.

So, I will close this PR with a reminder to add a warning on _classes/mcPy/MultiProcessing.py_

**Pull request closed without merging**